core/modules/book/book.install

```diff
@@ -138,6 +138,15 @@ function book_update_1002() {
   $config->save();
 }
 
+/**
+ * Add option for dis-allowing books for all content types.
+ */
+function book_update_1003() {
+  $config = config('book.settings');
+  $config->set('book_allowed_all', 1);
+  $config->save();
```

Comment:
I always like to do a value check to prevent issues if the update is double-run:

```suggestion
  if ($config->get('book_allowed_all') === NULL) {
    $config->set('book_allowed_all', 1);
    $config->save();
  }
```